src/cfile/writer.py

```diff
@@ -762,3 +772,12 @@ def _write_endif_directive(self, elem: core.EndifDirective) -> None:
     def _write_extern(self, elem: core.Extern) -> None:
         self._write(f'extern "{elem.language}"')
         self.last_element = ElementType.DIRECTIVE
+
+    def _write_condition(self, elem:core.Condition) -> None:
+        if elem.type == core.ConditionType.IF:
+            self._write(f"if ({elem.condition})")
```

Comment:
You need to add support for new formatting styles in style.py then support it in writer.py

- Space after if but before parenthesis? `if(` vs. `if (`
- Space around the condition? `(condition)` vs. `( condition )`